Sources/PostgresNIO/New/Data/Int+PSQLCodable.swift

```diff
@@ -0,0 +1,167 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 12.01.21.
+//
+
+extension UInt8: PSQLCodable {
```

Comment:
Ditch Xcode template headers.
```suggestion
extension UInt8: PSQLCodable {
```